HLSPlugin/src/com/kaltura/hls/m2ts/M2TSFileHandler.as

```diff
@@ -1,17 +1,19 @@
 package com.kaltura.hls.m2ts
 {
+	import com.kaltura.hls.HLSIndexHandler;
 	import com.kaltura.hls.HLSStreamingResource;
 	import com.kaltura.hls.SubtitleTrait;
 	import com.kaltura.hls.manifest.HLSManifestEncryptionKey;
 	import com.kaltura.hls.muxing.AACParser;
 	import com.kaltura.hls.subtitles.SubTitleParser;
 	import com.kaltura.hls.subtitles.TextTrackCue;
-	import com.kaltura.hls.HLSIndexHandler;
 	
 	import flash.utils.ByteArray;
 	import flash.utils.IDataInput;
 	import flash.utils.getTimer;
 	
+	import mx.utils.Base64Encoder;
```

Comment:
Does this bring in any weird dependencies? Casual inspection shows older Flex has this as a self-contained class.